GramAddict/core/views.py

```diff
@@ -657,6 +657,37 @@ def navigateToLanguage(self):
 
         return LanguageView(self.device)
 
+    def changeToUsername(self, username):
+        action_bar = self.device.find(resourceId=ResourceID.ACTION_BAR_LARGE_TITLE)
+        current_profile_name = action_bar.get_text().upper()
+        if current_profile_name == username.upper():
+            logger.info(
+                f"You are already logged as {username}!",
+                extra={"color": f"{Style.BRIGHT}{Fore.BLUE}"},
+            )
+            return True
+        if action_bar.exists():
+            action_bar.click()
+            random_sleep()
+            found_obj = self.device.find(
+                resourceId=ResourceID.ROW_USER_TEXTVIEW,
+                textMatches=case_insensitive_re(username),
+            )
+            if found_obj.exists():
+                logger.info(
+                    f"Switching to {configs.args.username}...",
```

Comment:
We don't need fetch from configs, `username` is enough.

Reply:
Blame lube. :D

Reply:
ops, that was stupid :D